client/src/index.jsx

```diff
@@ -77,10 +77,10 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    axios.get('/products?page=1&count=1')
+    axios.get(`/products/${this.props.init.id}`)
```

Comment:
I think these API calls might be were tests were broken. No biggie for now if we deprecate the index.jsx tests, but for future testing my suppression of these calls and checking provided data is necessary. That, or injecting a mock API function call. Those are possible, but it looked more involved than I was willing to get that early in our development when I wrote the tests!